web/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -26,6 +26,21 @@
                         <i class="fa fa-remove" aria-hidden="true"></i>
                     </button>
                 </div>
+                <div class="no-streams-to-show">
+                    <div class="subscribed_streams_tab_empty_text">
+                        <span>
+                            {{t 'You are not subscribed to any streams.'}} <a href="#streams/all">{{t 'View all streams'}}</a>
+                        </span>
+                    </div>
+                    <div class="all_streams_tab_empty_text">
+                        <span>
+                            {{t 'There are no streams you can view in this organization.'}}
+                            {{#if can_create_streams}}
+                                <a class="create_new_stream_link" href="#streams/new">{{t 'Create a stream'}}</a>
```

Comment:
I think there is no need for this class since the click handler for it was removed earlier.